### PR TITLE
bump to 4.1.0

### PR DESCRIPTION
**Pull request by rashmi43**
4.1 version for 4.1 release

### CODE REVIEW

**Review by Tirumalavasa — APPROVED** [review]
/lgtm

File: deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml [hunk 1]
```diff
@@ -0,0 +1,767 @@
...
+                "requests": {
+                  "cpu": "50m",
+                  "memory": "150Mi"
+                }
+              }
+            },
+            "initMongodb": {
+              "imageName": "icp-platform-auth",
+              "imageRegistry": "quay.io/opencloudio",
+              "imageTag": "3.10.0",
```
Comment (ujjwalchk-it) [inline deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml:124]: Should be 4.1.0 ?

File: deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml [hunk 2]
```diff
@@ -0,0 +1,767 @@
...
+    ## Supported platforms
+
+     - Red Hat OpenShift Container Platform 4.3 or newer installed on one of the following platforms:
+
+        - Linux x86_64
+        - Linux on Power (ppc64le)
+        - Linux on IBM Z and LinuxONE
+
+    ## Operator versions
+     - 4.1.0
```
Comment (ujjwalchk-it) [inline deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml:391]: We also should be having 4.0.0

**Review by ujjwalchk-it — COMMENTED** [review]
operator.yml also should have the changes https://github.com/IBM/ibm-iam-operator/blob/master/deploy/operator.yaml#L41

File: deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml (hunk 1 above)
Comment (Tirumalavasa) [reply deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml:124]: @ujjwalchk-it , will take care in another PR. this is just alm-examples content. so no issues.

File: deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml (hunk 2 above)
Comment (Tirumalavasa) [reply deploy/olm-catalog/ibm-iam-operator/4.1.0/ibm-iam-operator.v4.1.0.clusterserviceversion.yaml:391]: will take care in another PR

**Pull request merged**